src/ServicePulse.Host/vue/src/router/routeLinks.ts

```diff
@@ -38,7 +38,7 @@ const monitoringLinks = (root: string) => {
   };
 };
 
-const baseUrl = window.defaultConfig.base_url;
+export const baseUrl = window.defaultConfig.base_url;
```

Comment:
We want the responsibility of creating links to be in `routeLinks.ts`, hence there shouldn't be a need to export this